package.json

```diff
@@ -48,5 +48,12 @@
     "cross-env": "^7.0.3",
     "patch-package": "^6.4.7",
     "playwright": "1.40.0"
+  },
+  "overrides": {
```

Comment:
Note to self, need to recreate the lockfile before merging to main for overrides to work (merging updates from other branches breaks that update)

Reply:
I backed out the override. Seems to behave inconsistently after merges so don't want to deal with it.